### ISSUE TITLE
Monitoring if Elasticsearch is Down

### ISSUE THREAD

**Issue by jamiegwatkin** [ewr/elasticsearch_exporter#6]
Hi there,

I'm trying to monitor my elasticsearch instance so that if it's down an alert is triggered.

I created this basic rule -
```
# Alert if elasticsearch is unreachable for >1 minute.
ALERT ElasticSearchDown
  IF elasticsearch_up == 0
  FOR 1m
  WITH {
    severity="page"
  }
  SUMMARY "Instance {{$labels.instance}} elasticsearch is down."
  DESCRIPTION "{{$labels.instance}} of job {{$labels.job}} has been down for more than 1 minutes."
```

But the issue I'm facing is that if I issue a shutdown using the following command -
```
 curl -XPOST 'http://localhost:9200/_cluster/nodes/_local/_shutdown'
```

Instead of the metrics reflecting that Elasticsearch is down - all the metrics disappear!

Is there a better way to monitor that Elasticsearch is running and shouldn't the metric 'elasticsearch_up' == 0 if the elasticsearch_exporter cannot reach Elasticsearch?

**Comment by ewr** [comment]
It should have been reported, but you're absolutely right that it wasn't getting there. I'm just pushing 0.2.1, which includes that fix and some missing metrics fixed in #7.

**Comment by jamiegwatkin** [comment]
Perfect, thanks!
